Libplanet.Types/Tx/TxInvoice.cs

```diff
@@ -109,6 +115,13 @@ public TxInvoice(ITxInvoice invoice)
             {
                 throw new ArgumentNullException(nameof(updatedAddresses));
             }
+            else if (maxGasPrice is null ^ gasLimit is null)
```

Comment:
```suggestion
           if (maxGasPrice is null ^ gasLimit is null)
```

`else if` in such clause might make unnecessary diff when we need to remove above `if` block.

Reply:
Done.